opendere/game.py

```
@@ -56,19 +60,29 @@ def _start_game(self):
         messages = list()
         self.phase, self.ticks = 0, None
         if len(self.users) <= 3:
-            messages.append((self.channel, f"there aren't enough players to start a {self.name} game. try again later."))
-            self.users = {}
+            messages.append((self.channel, f"there aren't enough players to start a game of opendere in {self.channel}. please try again later."))
+            self.__init__(channel=None, bot=None, name=None)
```

I think we should raise an Exception here

```
class InsufficientPlayersError(ValueError):
    pass
```

Then the bot can catch the exception.

---

and when `opendere-sopel.py` catches the exception, it can `del game`